bootstrap/bootstrapper.yaml

```diff
@@ -52,17 +52,17 @@ spec:
     spec:
       containers:
       - name: kubeflow-bootstrapper
-        image: gcr.io/kubeflow-images-public/bootstrapper:v20180618-715aafc
+        image: gcr.io/kubeflow-images-public/bootstrapper-minikube:v20180813-cbc1c06-dirty-b82a7f
         workingDir: /opt/bootstrap
         command: [ "/opt/kubeflow/bootstrapper"]
         args: [
           "--in-cluster",
           "--namespace=kubeflow",
-          "--apply",
+          "--apply"
```

Comment:
Looks like the comma was removed by accident?

Reply:
The following line is commented out so remove this one as well